conf/computerome.config

```diff
@@ -0,0 +1,29 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'Computerome 2.0 cluster profile provided by nf-core/configs.'
+  config_profile_contact = 'Marc Trunjer Kusk Nielsen (@marcmtk)'
+  config_profile_url = 'https://www.computerome.dk/'
+  project = null
+  schema_ignore_params = "project,genomes,cache_dir,modules"
```

Comment:
`genomes` and `modules` should be ignored in the pipeline `nextflow.config` and so don't need to be set here?

Reply:
Like what you have done with `cache_dir` and `project` though 👌🏽